modules/service/src/main/resources/app.conf

```diff
@@ -68,6 +68,8 @@ app {
 
   # Authentication / Authorization settings
   auth {
+    test-mode = false # Set 'true' to bootstrap ACLs by granting top-level admin rights to anonymous users
+    test-mode = ${?IAM_TESTMODE}
     admin-groups = []
     admin-groups += ${?IAM_ADMINGROUP}
```

Comment:
I would add the option to add several groups, like this:
`admin-groups += ${?IAM_ADMINGROUP_1}`
`admin-groups += ${?IAM_ADMINGROUP_2}`
`admin-groups += ${?IAM_ADMINGROUP_3}`
`admin-groups += ${?IAM_ADMINGROUP_4}`